### ISSUE TITLE
Namespacing?

### ISSUE THREAD

**Issue by andystopia** [djmars-hall/CircutBoardTowerDefense#6]
I understand that we still have a small codebase, so it isn't that necessary *right now*; however, perhaps it would be more egronomic to work with if we had namespacing?

What do you guys think?

**Comment by djmars-hall** [comment]
I agree with doing namespaces. You don’t have to namespace everything tho
so we can just start with you doing that. And perhaps edit later

**Comment by andystopia** [comment]
That sounds pretty good to me. You can already see me doing a few namespaces for some things. My idea is having a new namespace for any object in the scene that has more than two scripts dedicated to it.

**Comment by djmars-hall** [comment]
That works for me.
